demos/multi_material/2d_cylindrical/2d_cylindrical.py

```diff
@@ -0,0 +1,320 @@
+# Idealised 2-D multi-material mantle convection problem inside an annulus
+# =
+
+# In this tutorial, we analyse mantle flow in a 2-D annulus domain whereby a denser and
+# more viscous layer, which could represent some lower-mantle structures on Earth, is
```

Comment:
Be a little more quantitative here -- how much denser, how much more viscous?

Reply:
Is it important at this stage? Perhaps I can spell it out where I define the material properties.